tests/std/tests/P0768R1_spaceship_operator/test.cpp

```diff
@@ -7,6 +7,11 @@
 #include <compare>
 #include <type_traits>
 
+// See https://github.com/microsoft/STL/pull/3581 for details

```

Comment:
We can just cite the PR with `GH-`. (The GitHub Pull Requests & Issues extension will provide a hover card in VSCode.)